src/Microsoft.ML.TimeSeries/AdaptiveSingularSpectrumSequenceModeler.cs

```diff
@@ -221,13 +221,16 @@ private static VersionInfo GetVersionInfo()
         {
             return new VersionInfo(
                 modelSignature: "SSAMODLR",
-                verWrittenCur: 0x00010001, // Initial
-                verReadableCur: 0x00010001,
+                //verWrittenCur: 0x00010001, // Initial
+                verWrittenCur: 0x00010002, // Added saving _state and _nextPrediction
```

Comment:
>saving _state [](start = 52, length = 13)

Why do you need to do it now? Is this for the stateful engine?

Reply:
So the preexisting test TestEstimatorCore() will pass on new timeseries estimators. One of the test steps in TestEstimatorCore() is to save and re-load transformer from disk. Without this fix saving and reloading of timeseries SSA transformers changes its prediction due to lost _state and _nextPrediction

---
In reply to: [229477366](https://github.com/dotnet/machinelearning/pull/1254#discussion_r229477366) [](ancestors = 229477366)